static/js/analytics.js

```diff
@@ -64,24 +76,51 @@ const composeStatisticsFromSingleUserJson = (json) => {
     .slice(0, 19)
     .replace("T", " ");
   const taggedFriends = Array.from(new Set(taggedFriendsFromJson(json)));
-  pageHtml += `<h2>${lastUserName}</h2>`;
-  pageHtml += `<p>Last fetched story: ${lastFetchedStoryTime}</p>`;
-  pageHtml += `<p>N: ${nStories} Stories | N: ${nPics} Pictures | N: ${nVideos} Videos</p>`;
-  pageHtml += `<canvas width="100%" id="chart-media"></canvas>`;
-  pageHtml += `<canvas width="100%" id="chart-hourly-freq"></canvas>`;
 
-  if (NGPSLocations > 0) {
-    pageHtml += `<p>Geotagged ${NGPSLocations} times</p>`;
+  const userDetails = {
+    "Username": lastUserName,
+    "Last fetched story": lastFetchedStoryTime,
+    "Number of stories": nStories,
+    "Number of pictures": nPics,
+    "Number of videos": nVideos,
+  };
+
+  let userDetailsBar = "";
```

Comment:
Same as suggested below. (btw pageHtml above is slightly different so i'd leave that unchanged)

Reply:
Done